packages/web-components/src/components/product-tile.ts

```diff
@@ -62,50 +62,52 @@ export class ProductTile extends LitElement {
             position: relative;
             text-decoration: inherit;
             text-size-adjust: none;
-            height: 100%;
-            font-family: Arial, Helvetica, sans-serif;
+            font-family: var(--relewise-font, Arial, Helvetica, sans-serif);
```

Comment:
Ønske: Bruge samme default som tailwind for det er en god default. Altså:
`ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";`